charts/etcd/values.yaml

```diff
@@ -39,13 +39,21 @@ backup:
   enableProfiling: false
   garbageCollectionPolicy: LimitBased
   maxBackups: 7
+  enableCompactionTempfs: false
   resources:
     limits:
       cpu: 100m
       memory: 512Gi
     requests:
       cpu: 50m
       memory: 128Mi
+  compactionResources:
+    limits:
+      cpu: 700m
+      memory: 3Gi
+    requests:
+      cpu: 500m
+      memory: 2Gi
```

Comment:
These values may have to be conditional on using _tmpfs_. Can you please refer to the latest results from the compaction performance tests? We will need to add at least `8Gi` for memory (so, `10Gi` in total) if _tmpfs_ is used.